src/lib/docker-utils.ts

```diff
@@ -27,6 +28,18 @@ interface RsyncApplyOptions {
 	retryInterval: number;
 }
 
+// TODO: Correctly export this from docker-toolbelt
+interface ImageNameParts {
+	registry: string;
+	imageName: string;
+	tagName: string;
+	digest: string;
+}
+
+// How long do we keep a delta token before invalidating it
+// (10 mins)
+const DELTA_TOKEN_TIMEOUT = 60 * 10 * 1000;
```

Comment:
I'd prefer
```suggestion
const DELTA_TOKEN_TIMEOUT = 10 * 60 * 1000;
```
to go M/S/ms instead of `S/M/ms`